include/acl_types.h

```diff
@@ -613,6 +613,18 @@ typedef struct {
                                 // constructor malloc related
     } host_pipe_info;
 
+    struct {
+      // Used for device global ops
+      size_t offset;
+      void *read_ptr;
+      const void *write_ptr;
+      bool blocking;
```

Comment:
Please correct me if this is wrong but I don't see `blocking` being used in `acl_read_device_global`/`acl_write_device_global`? Would it be okay to not keep `blocking` in `device_global_info`?

Reply:
Hi Sophie, you are absolutely right. I have removed it.